src/crimson/os/seastore/omap_manager/btree/btree_omap_manager.cc

```diff
@@ -112,10 +112,11 @@ BtreeOMapManager::omap_get_value(
   LOG_PREFIX(BtreeOMapManager::omap_get_value);
   DEBUGT("key={}", t, key);
   return get_omap_root(
-    get_omap_context(t, omap_root.hint),
+    get_omap_context(t, omap_root.hint, omap_root.get_type()),
```

Comment:
Nit, this commit is somewhat confusing to follow because:
1. The actual change to interface `get_omap_context(t, hint, type)` happens after this commit.
2. The user of new interface `omap_root_le_t::get_omap_root(type)` (proving why the new interface is necessary) isn't with this commit.